src/main/java/com/dream11/aerospike/factory/AerospikeClientFactoryImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.dream11.aerospike.factory;
+
+import com.aerospike.client.Host;
+import com.aerospike.client.policy.ClientPolicy;
+import com.dream11.aerospike.client.AerospikeClient;
+import com.dream11.aerospike.client.AerospikeClientImpl;
+import com.dream11.aerospike.config.AerospikeConnectOptions;
+import com.dream11.aerospike.util.PolicyUtil;
+import com.dream11.aerospike.util.SharedDataUtils;
+import io.vertx.core.*;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class AerospikeClientFactoryImpl implements AerospikeClientFactory {
```

Comment:
Can we get rid of this class and do all this is `AerospikeClientImpl.java` itself